src/webui/www/public/scripts/client.js

```diff
@@ -348,6 +350,16 @@ window.addEvent('load', function () {
                         updateCategoryList();
                         torrentsTableContextMenu.updateCategoriesSubMenu(category_list);
                     }
+
+                    // re-select the previously selected rows
+                    if (full_update) {
+                        if (torrentsTableSelectedRows.length)
```

Comment:
I know this is kind of moot, but I still wish it can be `if (torrentsTableSelectedRows.length > 0)`

Reply:
That was bugging me a bit too. On one hand it works, but on the other it may not be immediately clear why. I'll update it to be more explicit.